apps/api/src/app/content-templates/usecases/compile-template/compile-template.usecase.ts

```diff
@@ -8,6 +8,21 @@ Handlebars.registerHelper('equals', function (arg1, arg2, options) {
   return arg1 == arg2 ? options.fn(this) : options.inverse(this);
 });
 
+Handlebars.registerHelper('titlecase', function (string) {
```

Comment:
While `string` is not a reserved word, some ide's might interpret this with wrong formatting. What do you think about switching this to str? or value? 
```suggestion
Handlebars.registerHelper('titlecase', function (value) {
```